nmodl/ode.py

```diff
@@ -12,11 +12,15 @@
 # import known_functions through low-level mechanism because the ccode
 # module is overwritten in sympy and contents of that submodule cannot be
 # accessed through regular imports
-known_functions = import_module('sympy.printing.ccode').known_functions_C99
+major, minor = (int(v) for v in sp.__version__.split(".")[:2])
+if major >= 1 and minor >= 7:
```

Comment:
This is not correct.

2.2 will not work here.

Reply:
Stupid me! thanks Nico for pointing that!

We should convert this to integer and compare against it (`major*10+minor > 17`).

Reply:
Only if minor will never be > 9